models/product-type/src/attribute-set-type/attribute-set-type-draft/generator.ts

```diff
@@ -0,0 +1,12 @@
+import { fake, Generator } from '@commercetools-test-data/core';
+import * as AttributeBooleanTypeDraft from '../../attribute-boolean-type/attribute-boolean-type-draft';
+import { TAttributeSetTypeDraft } from '../types';
+
+// https://docs.commercetools.com/api/projects/productTypes#attributesettype
+
+export const generator = Generator<TAttributeSetTypeDraft>({
+  fields: {
+    name: 'set',
+    elementType: fake(() => AttributeBooleanTypeDraft.random()),
```

Comment:
Just for my own understanding, are we placing this in as a default value? Assuming it could be any of them correct?

Reply:
Yessir. I wasn't sure how to do "any of these available attribute types"